python/hail/table.py

```diff
@@ -2129,19 +2127,19 @@ def globals(self) -> 'StructExpression':
 
     @property
     def row(self) -> 'StructExpression':
-        """Returns a struct expression including all row-indexed fields.
+        """Returns a struct expression of all row-indexed fields, including keys.
 
         Examples
         --------
         The data type of the row struct:
 
         >>> table1.row.dtype
-        dtype('struct{ID: int32, HT: int32, SEX: str, X: int32, Z: int32, C1: int32, C2: int32, C3: int32}')
```

Comment:
this is an example so I don't much care, but it seems like `Table.row` didn't change so the behavior here shouldn't change?

Reply:
yeah, definitely, given the example in row_value below.